packages/react-components/src/templates/__tests__/DiscoverPageBody.test.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { ComponentProps } from 'react';
+import { render } from '@testing-library/react';
+
+import DiscoverPageBody from '../DiscoverPageBody';
+
+const props: ComponentProps<typeof DiscoverPageBody> = {
+  aboutUs: '<h3>About us content</h3>',
+  pages: [
+    {
+      path: '/',
+      title: 'Page 1 Title',
+      text: 'Page 1 Text',
+    },
+    {
+      path: '/',
+      title: 'Page 2 Title',
+      text: 'Page 2 Text',
+    },
+  ],
+  members: [],
+};
+
+it('renders multiple page cards', () => {
```

Comment:
These test cases confuse me a bit, could you go through them and make sure the assertions are exactly what the tests titles say and the other way around :sweat_smile: